src/lxc/lxccontainer.c

```diff
@@ -2320,7 +2322,12 @@ static bool remove_from_array(char ***names, char *cname, int size)
 {
 	char **result = get_from_array(names, cname, size);
 	if (result != NULL) {
-		free(result);
+		int i;
+		for (i = 0; (*names)[i] != *result && i < size; i++) {
```

Comment:
You should be able to easily get the index with `size_t i = result - *names` instead of searching for it.

Reply:
Done